pkg/instrumentation/javaagent.go

```diff
@@ -10,11 +10,17 @@ import (
 )
 
 const (
-	envJavaToolsOptions   = "JAVA_TOOL_OPTIONS"
-	javaJVMArgument       = " -javaagent:/otel-auto-instrumentation-java/javaagent.jar"
-	javaInitContainerName = initContainerName + "-java"
-	javaVolumeName        = volumeName + "-java"
-	javaInstrMountPath    = "/otel-auto-instrumentation-java"
+	envJavaToolsOptions       = "JAVA_TOOL_OPTIONS"
+	javaJVMArgument           = " -javaagent:/otel-auto-instrumentation-java/javaagent.jar"
+	javaInitContainerName     = initContainerName + "-java"
+	javaVolumeName            = volumeName + "-java"
+	javaInstrMountPath        = "/otel-auto-instrumentation-java"
+	javaInstrMountPathWindows = "\\otel-auto-instrumentation-java"
+)
+
+var (
+	commandLinux   = []string{"cp", "/javaagent.jar", javaInstrMountPath + "/javaagent.jar"}
+	commandWindows = []string{"CMD", "/c", "copy", "javaagent.jar", javaInstrMountPathWindows}
```

Comment:
Just curious is there a reason why we use CMD /c instead of [Copy-Item](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/copy-item?view=powershell-7.4)?

Reply:
windows nanoserver doesn't come with powershell so application pods wouldn't be able to run Copy-Item directly:

```
    Command:
      Copy-Item
      javaagent.jar
      \otel-auto-instrumentation-java
    State:          Waiting
      Reason:       RunContainerError
    Last State:     Terminated
      Reason:       StartError
      Message:      failed to start containerd task "a8f40a5dc034e4a566e577b42d0e86cd8d9e96ca70817c696b6a57bd5f32d39b": hcs::System::CreateProcess a8f40a5dc034e4a566e577b42d0e86cd8d9e96ca70817c696b6a57bd5f32d39b: The system cannot find the file specified.: unknown
      Exit Code:    128
...

Reply:
Updated the description to not say powershell, but just cmd.exe